src/events.d.ts

```diff
@@ -165,3 +165,6 @@ type FetchEventsResponse = {
   events: HistoryEventWithId[];
   eventGroups: CompactEventGroups;
 };
+
+type IterableEvent = HistoryEventWithId | CompactEventGroup;
```

Comment:
One other spot we could use the IterableEvent type is in getEventsInCategory() in get-event-categorization.ts.

Reply:
Good catch! Just updated it!